### PR TITLE
Feat/swerve

### PR DESCRIPTION
**Pull request by JayK445**
Implement Phoenix 6 swerve builder, should function very similarly (for basic drive, not sure about odometry, have to look into that)
~~In order to get rid of unrelated errors to work on swerve I... very messily commented out/deleted the offending files. So, anything related to pathplanner had been commented out and it's very messy (like GOOD GOD is it messy). Should I just delete those files so later pathplanner developments can just fill them in?~~
Deleted all pathplanner-related files, left a blank-ish slate for future development
Also not entirely sure about how good the swerve implementation is.

### CODE REVIEW

**Comment by JayK445** [comment]
don't merge yet. going to delete the jank pathplanner stuff; leave empty slate for future dev

**Review by audreypj — COMMENTED** [review]
Changes look good but I think because you did swerve related stuff we should wait until it is tested on Saturday to merge into dev. I merged Derek and I's changes today which were really just removing pp errors (so mostly deleting stuff, you basically have the same changes here). So dev should be pretty good rn, and we can use this branch on Sat for swerve testing.

**Review by audreypj — APPROVED** [review]
Tested this code today and it functions, swerve values need to be tuned but this code can be merged into dev

**Pull request merged**
